pramen-py/src/pramen_py/runner/runner_transformation.py

```diff
@@ -67,16 +69,25 @@ def overwrite_info_dates(
     return config
 
 
-def discover_transformations(
-    path: pathlib.Path = TRANSFORMATIONS_DIR,
-) -> Generator[Type[T_TRANSFORMATION], None, None,]:
+def get_transformations_dir(dir_key: str, envs: Env) -> pathlib.Path:
```

Comment:
After all my notes I see you just need to do something like:
```python
return env.path(dir_key, DEFAULT_PATH)
```

this is it. I think you don't need a function for this, just put it directly into load_modules

Reply:
and nothing to test here